packages/fiori/src/Wizard.js

```diff
@@ -0,0 +1,650 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { fetchI18nBundle, getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { getEffectiveAriaLabelText } from "@ui5/webcomponents-base/dist/util/AriaLabelHelper.js";
+import ItemNavigation from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import NavigationMode from "@ui5/webcomponents-base/dist/types/NavigationMode.js";
+import Float from "@ui5/webcomponents-base/dist/types/Float.js";
+import ResizeHandler from "@ui5/webcomponents-base/dist/delegate/ResizeHandler.js";
+import { isPhone } from "@ui5/webcomponents-base/dist/Device.js";
+
+// Texts
+import {
+	WIZARD_NAV_ARIA_ROLE_DESCRIPTION,
+	WIZARD_NAV_STEP_DEFAULT_HEADING,
+} from "./generated/i18n/i18n-defaults.js"
+
+// Step in header and content
+import WizardTab from "./WizardTab.js";
+import WizardStep from "./WizardStep.js";
+
+// Template and Styles
+import WizardTemplate from "./generated/templates/WizardTemplate.lit.js";
+import WizardCss from "./generated/themes/Wizard.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-wizard",
+	managedSlots: true,
+	properties: /** @lends sap.ui.webcomponents.fiori.Wizard.prototype */ {
+		/**
+		 * Defines the width of the component.
+		 * @private
```

Comment:
Do we need this property? I think we only need to check the breakpoint for phone?

Reply:
Not needed indeed, previously I did not have the width property and used the phoneMode to invalidate the component.